tensorflow/compiler/xla/debug_options_parsers_test.cc

```diff
@@ -26,8 +26,8 @@ namespace xla {
 
 // Test that the xla_backend_extra_options flag is parsed correctly.
 TEST(DebugOptionsFlags, ParseXlaBackendExtraOptions) {
-  std::unordered_map<string, string> test_map;
-  string test_string = "aa=bb,cc,dd=,ee=ff=gg";
+  std::unordered_map<std::string, std::string> test_map;
+  std::string test_string = "aa=bb,cc,dd=,ee=ff=gg";
```

Comment:
While this change is correct, if required, it should be done in a separate dedicated PR.

Reply:
this change, and others below are all required to make the corresponding testcases build in ROCm mode, they will all fail compilation without the change.

The changes do not seem ROCm specific, so am not convinced whether these tests are working to begin

Reply:
compile errors without the fix

```
...
external/com_google_googletest/googletest/include/gtest/internal/gtest-port.h:881:23: note:   'testing::internal::string'
 typedef ::std::string string;
                       ^
tensorflow/compiler/xla/debug_options_parsers_test.cc:29:33: error: 'test_map' was not declared in this scope
   unordered_map<string, string> test_map;
                                 ^
tensorflow/compiler/xla/debug_options_parsers_test.cc:30:10: error: expected ';' before 'test_string'
   string test_string = "aa=bb,cc,dd=,ee=ff=gg";
...
  ```